wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -162,6 +163,7 @@ def run(
                 launch_project,
                 entry_point,
             )
+            launch_project.fill_macros(image)
```

Comment:
Why not jus do this once outside the if statement?

Reply:
u r smart!